sanic/sanic.py

```diff
@@ -296,18 +295,11 @@ def run(self, host="127.0.0.1", port=8000, debug=False, before_start=None,
         :param sock: Socket for the server to accept connections from
         :param workers: Number of processes
                         received before it is respected
-        :param loop: asyncio compatible event loop
         :param protocol: Subclass of asyncio protocol class
         :return: Nothing
         """
         self.error_handler.debug = debug
         self.debug = debug
-        if loop is not None:
```

Comment:
Basically, with this PR we remove the loop functionality. But we keep the `loop` param but it will do nothing except trigger the warnings message so that anyone who was using the `loop` param won't just get a `TypeError` Exception and not know why. Then at the next big release say `0.4.0` we remove the param since we gave people enough of a heads up.

A lot of people just install sanic with `pip install sanic` so not having enough warning can potentially kill a lot of the apps people have built.

TL;DR

```python
if loop is not None:
    warnings.warn((
        '`loop` parameter will be removed after '
        'v`0.3.0` make sure to remove it before the next release!'))
```